static/styles.css

```diff
@@ -91,13 +113,13 @@ img.soil {
     left: 5%;
     bottom: 5px;
 }
-p.growingImg {
-    position: absolute;
-    width: 115px;
-    height: 115px;
-    top: 20px;
-    left: 33px;
-    background-repeat: no-repeat;
```

Comment:
I'm not sure it's this change, but I'm now seeing scrollbars in my growing plots.